tests/SprykerEcoTest/Zed/Heidelpay/Business/Facade/ExecuteDebitOnRegistrationTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Heidelpay\Business;
+
+use Generated\Shared\Transfer\PaymentHeidelpayTransactionLogCriteriaTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use SprykerEco\Shared\Heidelpay\HeidelpayConfig;
+
+/**
+ * @group Functional
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Heidelpay
+ * @group Business
+ * @group Facade
+ * @group ExecuteDebitOnRegistrationTest
+ */
+class ExecuteDebitOnRegistrationTest extends HeidelpayPaymentTest
+{
+    /**
+     * @return void
+     */
+    public function testSuccessfulExecuteDebitOnRegistration(): void
+    {
+        //Arrange
```

Comment:
I would recommend to add spacing after //
``// Arrange`
same for introduced comments below